phpunit

```diff
@@ -12,7 +12,7 @@ if (!getenv('SYMFONY_PHPUNIT_VERSION')) {
         if (false === getenv('SYMFONY_PHPUNIT_REMOVE_RETURN_TYPEHINT') && false !== strpos(@file_get_contents(__DIR__.'/src/Symfony/Component/HttpKernel/Kernel.php'), 'const MAJOR_VERSION = 3;')) {
             putenv('SYMFONY_PHPUNIT_REMOVE_RETURN_TYPEHINT=1');
         }
-        putenv('SYMFONY_PHPUNIT_VERSION=8.3');
+        putenv('SYMFONY_PHPUNIT_VERSION=8.5');
```

Comment:
We're not required to upgrade, isn't it? we could then revert the bump in FWB's composer.json.
Would be better IMHO, more seamless.

Reply:
Since the `match` construct was added to php 8, PHPUnit < 9.2 is incompatible. We are required to upgrade to PHPUnit 9.3, if we want to run our test suite on php 8. Reverting that line would make the PR mergable though.

Reply:
Please revert for now so we can merge what is mergeable